packages/ember-glimmer/lib/index.js

```diff
@@ -1,3 +1,85 @@
+/**
+
+ [Glimmer](https://github.com/tildeio/glimmer) is a [Handlebars](http://handlebarsjs.com/)
```

Comment:
Updated the documentation to point at glimmer instead of htmlbars.  Combined the ember-htmlbars and ember-templates documentation into just ember-glimmer.  Need some confirmation that this is what is desired